site/static/examples/ingress/usage.yaml

```diff
@@ -51,7 +51,7 @@ kind: Ingress
 metadata:
   name: example-ingress
   annotations:
-    ingress.kubernetes.io/rewrite-target: /
+    nginx.ingress.kubernetes.io/rewrite-target: /$2
```

Comment:
You can't use $2 if you don't have groups defined in the path

Reply:
So I should rather keep it `/`?